bower.json

```diff
@@ -2,34 +2,35 @@
   "name": "origin-web-console",
   "version": "1.3.0-alpha",
   "dependencies": {
-    "angular": "1.3.20",
+    "angular": "1.5.11",
     "json3": "3.3.2",
     "es5-shim": "3.1.1",
-    "angular-resource": "1.3.20",
-    "angular-cookies": "1.3.20",
-    "angular-sanitize": "1.3.20",
-    "angular-animate": "1.3.20",
-    "angular-touch": "1.3.20",
-    "angular-route": "1.3.20",
+    "es5-dom-shim": "*",
```

Comment:
We effectively already were since that is what hawtio-core was asking for before, there is not other option as the es5-dom-shim repo doesn't actually have releases.

We can probably drop this dependency altogether now, I don't think any of the browsers we support now even need this.  But I would rather do that independent of this set of changes so we aren't trying to figure out what caused the issue if something ends up breaking.